homeassistant/components/mcp_server/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""The Model Context Protocol Server integration."""
+
+from __future__ import annotations
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from . import http
+from .const import DOMAIN
+from .session import SessionManager
+from .types import MCPServerConfigEntry
+
+__all__ = [
+    "DOMAIN",
+]
+
+PLATFORMS: list[Platform] = []
+
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Model Context Protocol component."""
+    http.async_register(hass)
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: MCPServerConfigEntry) -> bool:
+    """Set up Model Context Protocol Server from a config entry."""
+
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
```

Comment:
we can drop the setting up, unloading and the `PLATFORMS` constant as they are not used.